index.html

```diff
@@ -691,6 +691,26 @@ <h2>Verifiable Credential</h2>
         </aside>
       </section>
     </section>
+
+    <section>
+      <h2>Securing JSON-LD</h2>
+      <section>
```

Comment:
```suggestion
      <p>Enabling the use of semantics through JSON-LD to describe credentials is highly desirable for many use cases. The following items cover common methods of allowing semantics to be present, but to utilize common signing approaches rather than requiring transformation of the content through URDNA2015.</p>
      <p>This means that the underlying n-quads are not signed, and the user should be aware of this, in the event that what they wish to sign is a semantic representation of the data, rather than the data itself.</p>
      <section>
```

Reply:
Can I resolve this comment, seems we have found a better place to state this.

Reply:
@OR13 — Can you provide a link to that better place/statement?

Reply:
+1 to a link ... and wherever that is, instead of saying "transformation of the content prior to signing" we should say "transformation of the content via canonicalization prior to signing" ... since the content will, in fact, be transformed prior to signing, but only by cryptographic hash.

We should also not say that the following items "cover methods" but rather "are methods"... wherever we're making these statements, so as not to preclude future / alternative methods.

Reply:
See this comment: https://github.com/w3c/vc-jwt/pull/51#issuecomment-1426782687

Basically, this document is not a good place to do this constraining, or talk about nquads, instead we should try to constrain the media type in the document that registers it... which is the core data model.

This document just refers to the core data model content types when securing them, best to not describe what data integrity proofs do here... since we are not using them here.

Reply:
On a related note, we talked about using this spec to "forbid proof" in the case the core data model allows it in `credential+ld+json`... 

I think it is best for JWT representations to treat the header and claim set as JSON, and not do anything specific to data integrity proofs or any other JSON or JSON-LD specific formats...

From the view of JWS, JSON can have members, and if they are not understood, they are ignored.

There should be no special "edge cases" related to data integrity proofs in VC-JWT, the perspective should be that they are just JSON, regardless of how many proof sets or chains they have.

The place to constrain `credential+ld+json` is this pull request or related pull requests in the core data model: 

https://github.com/w3c/vc-data-model/pull/1014